resources/lib/oeWindows.py

```diff
@@ -696,7 +696,10 @@ def onClick(self, controlID):
                     self.visible = False
                     self.close()
                     if lang_new:
-                        xbmc.executebuiltin(f'SetGUILanguage({str(lang_new)})')
+                        if xbmc.getCondVisibility(f'System.HasAddon({lang_new})') == True:
```

Comment:
There is a race with the language being loaded in the background. We have to wait to complete.
```suggestion
                        for _ in range(20):
                            if xbmc.getCondVisibility(f'System.HasAddon({lang_new})'):
                                break
                            oe.xbmcm.waitForAbort(0.5)
                        if xbmc.getCondVisibility(f'System.HasAddon({lang_new})') == True:
```